dialogy/plugins/text/intent_entity_mutator/actions_on_primitives.py

```diff
@@ -0,0 +1,27 @@
+"""
+    This module consist of custom functions that operate on primitives such as transcripts, intents and entities
+
+"""
+
+from dialogy.types import BaseEntity, Intent
+from typing import List
+import copy
+
+# The below custom functions are coming in from ashley's StateToIntent code wherein we need to count the number of digits and check if a number is present in more than 50% of transcripts
```

Comment:
I don't think we should add this information as a comment. Dialogy is a public repo 😬